.github/workflows/deploy-prep.yml

```diff
@@ -0,0 +1,20 @@
+name: Deployment Prep
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+jobs:
+  config:
+    if: github.repository == ‘anuraghazra/github-readme-stats’
```

Comment:
These quotes looks a bit odd lol 

![image](https://user-images.githubusercontent.com/35374649/213918522-cacce8b2-bd9a-4f58-b5d8-865f3cec8812.png)

Reply:
Copied it from StackOverflow 🤦 Fixed it now!